src/sdk/iland.ts

```diff
@@ -3,20 +3,25 @@ import { Logger } from './logger/logger';
 import { Http } from './service/http/http';
 import { BasicConfiguration } from './config/basic-configuration';
 import { noop } from 'rxjs/util/noop';
+import { ImpersonatedRole } from './model/iam/role/__json__/role-json';
 
 const DEFAULT_API_URL = `${BasicConfiguration.getApiUrl()}/v1`;
 
 export abstract class Iland {
 
   public static baseUrl: string;
 
+  public static impersonatedRole: ImpersonatedRole | undefined;
+
   private static _authProvider: AuthProvider | undefined;
 
   private static _logger: Logger;
 
   private static _http: Http | undefined;
 
-  static init(_authProvider: AuthProvider, _config?: IlandSdkConfig, _logger?: Logger) {
+  static init(_authProvider: AuthProvider, _config?: IlandSdkConfig, _logger?: Logger,
```

Comment:
this should not be passed in as global SDK configuration, instead you should simply have methods on the IlandBrowserAuthProvider for `testRole(roleUuid: string)` and `endRoleTest()` ( no need to put this on the direct grant one right now)

Reply:
ok for the testRole() method, but since we are refreshing the whole app do we need a endRoleTest method ? we won't use use it.

Reply:
the refresh is an implementation detail of the console UI and is not related to the implementation of the underlying feature in this SDK

Reply:
ok then. i'll add it though.